client/src/components/SignUpForm.js

```diff
@@ -1,77 +1,109 @@
-            required={true}
-            autoComplete="off"
-            tabIndex="3"
-            aria-invalid="false"
-            aria-controls="password"
-            aria-label="Choose a strong password."
-            aria-activedescendant="password-item-0"
-          />
-          <label htmlFor="passphrase" className="d-none">
-            Passphrase
-          </label>
-        </div>
-        <div>
-          <input
-            type="submit"
-            value="Sign up"
-            tabIndex="4"
-            className="btn btn-black btn-subscribe btn-block"
-          />
-        </div>
-      </form>
-    );
+function SignUpForm({ history }) {
+  const [loading, setLoading] = useState();
+  const dispatch = useDispatch();
+  const { values, handleChange, handleSubmit } = useForm(
+    { email: '', password: '' },
+    postSignUp
+  );
+
+  async function postSignUp() {
+    try {
+      setLoading(true);
+      const { data } = await axios.post(
+        `${process.env.REACT_APP_EXPRESS_SERVER}/signup`,
+        values
+      );
+      
+      const res = await data;
+
+      if (res.error) {
+        console.error(data.error);
+        return history.push('/signup-error');  
+      }
+
+      if (res.token) {
+        dispatch({
+          type: actions.SET_AUTH_USER,
+          authUser: {
+            email: values.email
+          }
+        });
+  
+        localStorage.setItem('token', res.token);
+  
+        setLoading(false);
+        return history.push('/');
+      }
+    } catch (error) {
+      setLoading(false);
+      console.error(error);
+    }
   }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className="form-group">
+        <label htmlFor="email" className="d-none">
+          Email address
+        </label>
+        <input
+          onChange={handleChange}
+          value={values.email}
+          name="email"
+          type="email"
+          id="email"
+          className="form-control mb-2"
+          placeholder="Email address"
+          required
+          autoComplete="off"
+          tabIndex="2"
+          autoCapitalize="none"
+          aria-invalid="false"
+          aria-controls="email"
+          aria-label="What is your email address?"
+          aria-activedescendant="email-item-0"
+        />
+        <label htmlFor="password" className="d-none">
+          Password
+        </label>
+        <input
+          onChange={handleChange}
+          value={values.password}
+          name="password"
+          type="password"
+          id="password"
+          className="form-control mb-2"
+          placeholder="Password"
+          required
+          autoComplete="off"
+          tabIndex="3"
+          aria-invalid="false"
+          aria-controls="password"
+          aria-label="Choose a strong password."
+          aria-activedescendant="password-item-0"
+        />
+        <label htmlFor="passphrase" className="d-none">
+          Passphrase
```

Comment:
When the user signs up for Universe, they do not have to choose a `Passphrase`, we will authenticate the user with only their email address and password.

Reply:
Same as previous comment 👆 Will remove it